### DISCUSSION TITLE
Unable to boot from USB

### DISCUSSION THREAD

**Discussion by mprattmd** [sebanc/brunch#1606] [General]
I have brunch installed on a Surface Go 1.  Works fine.  I am trying to boot from a USB to do anything, and I am simply unable to.  I have tried booting from Fedora, ChromeOS Flex, and Windows to no avail.  
I am very familiar with linux and windows and am stumped. 

I am trying to edit the grub config, but crosh is not letting me enter shell.

I do not see the brunch bootloader at all, it goes straight to chrome so I am unable to try keyboard commands or anything

Any suggestions?

**Comment by tejasraman** [comment]
Press and hold volume down when you are clicking on the power button (while USB is plugged in).

See here: [Boot Surface from a USB device - Microsoft Support](https://support.microsoft.com/en-us/surface/boot-surface-from-a-usb-device-fe7a7323-8d1d-823d-be17-9aec89c4f9f5)

**Comment by mprattmd** [comment]
I finally got it to boot a windows 11 usb
I couldn’t get any linux to boot at all


From: tejasraman ***@***.***>
Date: Tuesday, August 9, 2022 at 12:01 PM
To: sebanc/brunch ***@***.***>
Cc: Mark Test ***@***.***>, Author ***@***.***>
Subject: Re: [sebanc/brunch] Unable to boot from USB (Discussion #1606)

Press and hold volume down when you are clicking on the power button (while USB is plugged in).

See here: [Boot Surface from a USB device - Microsoft Support)[https://support.microsoft.com/en-us/surface/boot-surface-from-a-usb-device-fe7a7323-8d1d-823d-be17-9aec89c4f9f5)

**Comment by tejasraman** [comment]
Did it work? (The article I sent)

**Comment by mprattmd** [comment]
Once it booted, I figured I could re-install windows if needed
I was just in a panic I couldn’t get anything to boot

Is there a way to force a stop at the brunch bootloader screen?


From: tejasraman ***@***.***>
Date: Tuesday, August 9, 2022 at 12:05 PM
To: sebanc/brunch ***@***.***>
Cc: Mark Test ***@***.***>, Author ***@***.***>
Subject: Re: [sebanc/brunch] Unable to boot from USB (Discussion #1606)

Did it work?

**Comment by tejasraman** [comment]
That should be the default behavior - 10s at the boot loader screen.

Are you using brunch grub or grub from some other linux distro?

**Comment by mprattmd** [comment]
Brunch grub

I get the windows logo, immediately followed by Chrome splash

From: tejasraman ***@***.***>
Date: Tuesday, August 9, 2022 at 12:09 PM
To: sebanc/brunch ***@***.***>
Cc: Mark Test ***@***.***>, Author ***@***.***>
Subject: Re: [sebanc/brunch] Unable to boot from USB (Discussion #1606)

That should be the default behavior - 10s at the boot loader screen.

Are you using brunch grub or grub from some other linux distro?

**Comment by tejasraman** [comment]
That's normal. That is the Surface BIOS logo. Windows = spinning dots (if you've ever used Win8/10, you'd know)

**Comment by tejasraman** [comment]
You should see a screen with "ChromeOS" and "ChromeOS (Settings)" on it

**Comment by mprattmd** [comment]
First screen I get to is the chrome login screen with the version top upper left

From: tejasraman ***@***.***>
Date: Tuesday, August 9, 2022 at 12:13 PM
To: sebanc/brunch ***@***.***>
Cc: Mark Test ***@***.***>, Author ***@***.***>
Subject: Re: [sebanc/brunch] Unable to boot from USB (Discussion #1606)

You should see a screen with "ChromeOS" and "ChromeOS (Settings)" on it
